# Assignment 6

To measure the psychological state of Americans, a psychology researcher (in 1929) devised
three questions put to 3443 Americans from different walks of life. He asked them whether they
were happy. They could answer only **yes** or **no** to each question. By today’s standards the study
is quite simple, and he did no formal statistical analysis of his data. The frequency data (number
of **yes** and **no** responses within each employment category) from this study are in the file
*Americandata.csv*.

---

You need to use the data and answer the question, that is, **Is the type of profession related to the happiness of American people?**
Report the **χ2** statistic and the *statistical significance of test*, in addition to the contingency table.
Also, explain the results and what they mean. What does the value of **χ2** statistic mean? Is it
good or bad? Is the test statistically significant?

In [7]:
import pandas as pd
import scipy.stats as stats

df = pd.read_csv('Americandata.csv')

contingency_table = df.drop(columns = "Happy")

chi2, p, dof, expected = stats.chi2_contingency(contingency_table, correction=False)

print(f"Chi2 = {chi2}")
print(f"P = {p:.3f}")
print(f"Dof = {dof}")
print(f"Expected = {expected}")

Chi2 = 936.1394782304601
P = 0.000
Dof = 7
Expected = [[ 683.71157713  170.92789428  102.55673657   71.7897156    68.37115771
    49.91094513   25.63918414   17.09278943]
 [1316.28842287  329.07210572  197.44326343  138.2102844   131.62884229
    96.08905487   49.36081586   32.90721057]]


In [3]:
df.head(3)

,Happy,Students,Laborers,Preachers,Physicians,Housewives,Teachers,Lawyers,Musicians
0,Yes,390,378,35,159,78,108,11,31
1,No,1610,122,265,51,122,38,64,19


Put your analysis in the empty `markdown` cell below.

The X2 (chi2) is 936.14 and is good! This shows statistical signifigance between career type and happiness. The P value is (essentially) 0 and that shows that the odds that this data came about randomly and is not really correlated as being very low.